aspnetcore/fundamentals/index.md

```diff
@@ -183,42 +153,12 @@ For more information, see <xref:fundamentals/host/hosted-services>.
 
 For more information, see <xref:fundamentals/httpcontext>.
 
-## WebSockets
-
-[WebSocket](https://wikipedia.org/wiki/WebSocket) is a protocol that enables two-way persistent communication channels over TCP connections. It's used for apps such as chat, stock tickers, games, and anywhere you desire real-time functionality in a web app. ASP.NET Core supports web socket scenarios.
-
-For more information, see <xref:fundamentals/websockets>.
-
-::: moniker range=">= aspnetcore-2.1"
-
-## Microsoft.AspNetCore.App metapackage
-
-The [Microsoft.AspNetCore.App](https://www.nuget.org/packages/Microsoft.AspNetCore.App/) metapackage simplifies package management.
-
-For more information, see <xref:fundamentals/metapackage-app>.
-
-::: moniker-end
-
-::: moniker range="= aspnetcore-2.0"
-
-## Microsoft.AspNetCore.All metapackage
-
-The [Microsoft.AspNetCore.All](https://www.nuget.org/packages/Microsoft.AspNetCore.All) metapackage for ASP.NET Core includes:
-
-* All supported packages by the ASP.NET Core team.
-* All supported packages by Entity Framework Core.
-* Internal and 3rd-party dependencies used by ASP.NET Core and Entity Framework Core.
-
-For more information, see <xref:fundamentals/metapackage>.
-
-::: moniker-end
-
-## .NET Core vs. .NET Framework runtime
-
-An ASP.NET Core app can target the .NET Core or .NET Framework runtime.
+<a name="download"></a>
 
-For more information, see [Choosing between .NET Core and .NET Framework](/dotnet/articles/standard/choosing-core-framework-server).
+## How to download a sample
```

Comment:
These sample download instructions feels a bit buried. Maybe this should go at the end of the Intro to ASP.NET Core topic?